### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by logankilpatrick**
Fixed the plot plan enabling/disabling description.

### CODE REVIEW

**Comment by asinghvi17** [comment]
Why not both?  It's an internal API, true, but it's way easier, and allows you to get stuff in the plot pane if you need to.

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Examples, and test infrastructure, are hosted at [`MakieGallery.jl`](https://git
 
 ## Using Juno with Makie
 
-The default OpenGL backend for Makie is not interactive in the Juno plotpane - it just shows a PNG instead.  To get full interactivity, you can disable the plotpane in Atom's settings, or execute `Atom.PlotPaneEnabled[] = false`.
+The default OpenGL backend for Makie is not interactive in the Juno plotpane - it just shows a PNG instead.  To get full interactivity, you can disable the plotpane in Atom's settings by going to `Juno` - `Settings` - `UI Options` - Then, make sure `Enable Plot Plane` is __not__ checked.
```
Comment (aminya) [inline README.md]:
```suggestion
, or execute `Atom.PlotPaneEnabled[] = false`.
```

**Comment by logankilpatrick** [comment]
Sebastian stated it should not be used which is why I suggested to remove it

**Comment by SimonDanisch** [comment]
By the way, I decoupled this from the plot pane being disabled or not with `AbstractPlotting.inline(false)`  - this will always open a window now... `AbstractPlotting.inline(true)` will only show something, when the plotpane is enabled, though.

File: README.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ Examples, and test infrastructure, are hosted at [`MakieGallery.jl`](https://git
...
```
Comment (asinghvi17) [inline README.md]:
```suggestion
The default OpenGL backend for Makie is not interactive in the Juno plotpane - it just shows a PNG instead.  To get full interactivity, you can run `AbstractPlotting.inline!(false).

If that fails, you can disable the plotpane in Atom's settings by going to `Juno` - `Settings` - `UI Options` - Then, make sure `Enable Plot Plane` is __not__ checked. 
```

Comment (asinghvi17) [reply README.md]: @logankilpatrick this should be gtm

**Comment by logankilpatrick** [comment]
Done!

**Pull request merged**